app/components/provider_interface/activity_log_event_component.rb

```diff
@@ -0,0 +1,91 @@
+module ProviderInterface
+  class ActivityLogEventComponent < ViewComponent::Base
+    include ViewHelper
+    attr_reader :event, :application_choice
+    delegate :changes, to: :event
+
+    ORIGINAL_OPTION_STATUSES = %w[awaiting_provider_decision rejected].freeze
+
+    def initialize(activity_log_event:)
+      @event = activity_log_event
+      @application_choice = activity_log_event.audit.auditable
```

Comment:
Would you mind doing a Law of Demeter job on this? Clients shouldn't know about `audit`, ie expose `activity_log_event.application_choice`?